FrameworkSystem/Client/ComponentInstaller.py

```diff
@@ -1679,18 +1679,27 @@ def setupSite( self, scriptCfg, cfg = None ):
       result = self.setupComponent( 'service', system, service, extensions, monitorFlag = False )
       if not result['OK']:
         gLogger.error( result['Message'] )
+      result = MonitoringUtilities.monitorInstallation( 'service', system, service )
```

Comment:
Should it be called even if setupComponent() call fails ? Same for other component types